hub/policy/src/main/java/uk/gov/ida/hub/policy/services/AuthnResponseFromCountryService.java

```diff
@@ -116,8 +116,8 @@ private ResponseAction nonSuccessResponse(SessionId sessionId) {
         return other(sessionId, false);
     }
 
-    private AttributeQueryRequest getAttributeQueryRequest(AttributeQueryContainerDto aqr) {
-        return new AttributeQueryRequest(aqr.getId(), aqr.getIssuer(), aqr.getSamlRequest(), aqr.getMatchingServiceUri(), aqr.getAttributeQueryClientTimeOut(), aqr.isOnboarding());
+    private AttributeQueryRequest getAttributeQueryRequest(AttributeQueryContainerDto aqr, InboundResponseFromCountry translatedResponse) {
```

Comment:
I don't think we need to pass the `InboundResponseFromCountry` in here. It gets passed to `new AttributeQueryRequest()` and included as a field on that object, but the important data is already wrapped up in the encrypted attribute-query in the `samlRequest` field, which is the thing the MSA will be looking at?

If we drop that it'll mean we don't need to make changes to the saml-soap-proxy `AttributeQueryContainerDto`. Or is there a good reason for including the `CountrySignedResponseContainer` in the dto?